syntax = "proto3";

package audio;

import "audio.proto";

service AudioService {
    rpc StreamTranscribe (stream AudioChunk) returns (TranscriptionResponse);
}